examples/multi_threading_context/main.rs

```diff
@@ -28,27 +28,28 @@ fn build_ui(application: &gtk::Application) {
     scroll.set_policy(gtk::PolicyType::Automatic, gtk::PolicyType::Automatic);
     scroll.add(&text_view);
 
-    let (tx, rx) = glib::MainContext::channel(glib::Priority::default());
+    let (tx, rx) = async_channel::unbounded();
```

Comment:
Should probably not be unbounded?

Reply:
Dunno... I do not like to use a value pulled out of this air if I do not understand the implications and these are just examples. Anyway I guess I will just change that to 10

Reply:
I think here it does not matter much, but any bounded value is usually better than it growing infinitely.